lib/matplotlib/__init__.py

```diff
@@ -660,12 +661,12 @@ class RcParams(MutableMapping, dict):
     --------
     :ref:`customizing-with-matplotlibrc-files`
     """
-
     validate = rcsetup._validators
 
-    # validate values on the way in
     def __init__(self, *args, **kwargs):
+        self._rcvalues = ChainMap({})
         self.update(*args, **kwargs)
+        self._rcvalues.new_child()
```

Comment:
If we want to keep an explicit reference to defaults, it'll have to be something like
```python
        self._rcvalues = ChainMap({})
        self.update(*args, **kwargs)
        self._rcvalues = self._rcvalues.new_child()
        self._defaults = self._rcvalues.maps[-1]
```
since we would want to let the defaults go through the validation step too. Does that look better?